backend/controllers/users.js

```diff
@@ -42,6 +42,21 @@ export const getOneUser = async (req, res) => {
   }
 }
 
```

Comment:
in line 40 it looks like there's a typo: `should be console.error(error.message)`